Tools/packages.config

```diff
@@ -7,7 +7,6 @@
   <package id="NUnit.Extension.NUnitV2ResultWriter" version="3.6.0" targetFramework="net461" />
```

Comment:
NUnit ConsoleRunner is directly launched by the build menu, and I guess other NUnit packages are dependencies.

Reply:
`NUnit.ConsoleRunner` does not have any dependencies, `NUnit.Console` has all those dependencies. As only `NUnit.ConsoleRunner` seems to be used I've only moved it into `packages.csproj`.

Reply:
My bad, the dependencies are still needed otherwise the tests won't run (with error: `Unknown result format: nunit2`). Adding `NUnit.Console` as the dependency fixed the issue.